jsons/parameters.json

```diff
@@ -16750,6 +17386,40 @@
         },
         "value_type": "bool"
     },
+    {
+        "default_value": true,
+        "description": "Either to use unimod4. TODO: this param need revisiting, once it is clear what the param stand for!",
+        "key_translations": {
+            "diann_style_1": "use_unimod4_<DROP_KEY>",
+            "ursgal_style_1": "use_unimod4"
```

Comment:
can unimod not be supplied to the algorithm? If not what about usermods? If so then we should not have a parameters for that but let the wrapper decide to use unimod4 if no unimod file has been applied, I guess.

Reply:
Both Andrew and me don't know what this param is actually doing. I added it for now as it is part of the command line. During testing, Andrew wanted to evaluate this and then update the param or even get rid of it. Hence, the 'TODO' in the description :)

Reply:
From the source it looks like this is a shorthand for adding unimods by accession, diann ships with the 'full unimod' ` --full-unimod loads the complete UniMod modification database and disables the automatic conversion of modification names to the UniMod format` there's also the documented --mod argument `--mod [name],[mass],[optional: 'label'] declares a modification name. Examples: "--mod UniMod:5,43.005814", "--mod SILAC-Lys8,8.014199,label"`.

I would be fine with dropping the shorthand and pushing mods through the command line arguments.